ingestion_tools/scripts/data_validation/standardized/tests/test_tiltseries.py

```diff
@@ -126,3 +127,79 @@ def test_mdoc_tiltseries_range(
         )
 
     ### END metadata-mdoc consistency tests ###
+
+    @allure.title("Tiltseries: sum of exposureDose of all frames associated with a tilt series == totalFlux +-1 of tilt series")
+    def test_exposure_dose(self, frame_metadata: Dict, tiltseries_metadata: Dict):
+        assert sum(f.get("exposure_dose", 0) for f in frame_metadata["frames"]) == pytest.approx(tiltseries_metadata["total_flux"], abs=1)
+
+    @allure.title("PerSectionParameters: number of frames >= # of per section parameters.")
```

Comment:
The label of what it is testing against like "PerSectionParameters"  is very neat! 😄